pkg/controllers/default-resources.go

```diff
@@ -28,14 +28,14 @@ import (
 
 const ReleaseImage = "quay.io/openshift-release-dev/ocp-release:4.9.15-x86_64"
 
-func ScafoldHostedCluster(hyd *hypdeployment.HypershiftDeployment, hcs *hyp.HostedClusterSpec) *hyp.HostedCluster {
+func ScaffoldHostedCluster(hyd *hypdeployment.HypershiftDeployment) *hyp.HostedCluster {
 
 	return &hyp.HostedCluster{
 		ObjectMeta: v1.ObjectMeta{
 			Name:      hyd.Name,
 			Namespace: hyd.Namespace,
```

Comment:
for this one, should we also consider using the `spec.targetNamespace`?

Reply:
But yes, I think we want the remote namespace to be targeted.

Reply:
Sounds good. @ianzhang366 will put a pull request in to address this.